pkg/util/fielderrors/fielderrors.go

```diff
@@ -1,207 +0,0 @@
-/*
```

Comment:
No cycle, and one of the followup PRs does in fact create a sub-pkg :)

No sweat that it is friday - I wanted to get these off my laptop before I broke it.